pyspades/vxl.pyx

```diff
@@ -118,16 +118,17 @@ cdef class VXLData:
 
     def destroy_point(self, int x, int y, int z):
         if not self.get_solid(x, y, z) or z >= 62:
-            return False
+            return 0
```

Comment:
I believe that this is to support the `total_blocks_removed` attribute. The function is currently used in `scripts/platform.py` in `if` statements, which is fine because 0 is falsey, and non-zero is truthy. Where it is used in `server.py` has been updated by infogulch to suit.